### DISCUSSION TITLE
How to add LORAs correctly?

### DISCUSSION THREAD

**Discussion by webstunner** [hako-mikan/sd-webui-regional-prompter#150] [Q&A] [answered]
Hi! I am confused at how to add multiple LORAs correctly. I want the LORAs to affect the whole render, while using regions to 'color' or describe the outfits in detail.

I have:
1girl, photorealistic, shirt, pants
AND <lora:lora1:.3> 
AND <lora:lora2:.2>
light denim, shirt BREAK 
black, leather pants, add belt  

Is that the right way? Seems like the LORAs are not applied correctly. Render looks distorted. While pants are correct, shirt is usually white or blue but not denim... so I am sure I got this wrong.

I have:
Regional Prompter ACTIVE
Generation mode ATTENTION (Latent always give me an error)
Base Ratio 0.2
Common prompt checked
Split Mode VERTICAL
Divide Ratio 1,1

**Comment by Symbiomatrix** [comment, marked as answer]
Hello.
If the loras should apply to the whole region, attention is correct.
AND is not recommended, though there's a conversion for it (latent only, I think?), it might throw off the structure. You'll want to place the loras normally in any region except for common. Use BREAKs or ADDROW / ADDCOL to separate regions.
The most likely issue is that you're missing an additional break (after shirt pants) - you haven't separated the light denim into a region after common, so there's just one, black leather, which receives the common clause, making it basically identical to no regions.

**Reply by webstunner** [reply]
Thank you for replying to my question. I'll try when I get a chance.
